src/app/modules/group/components/joined-group-list/joined-group-list.component.ts

```diff
@@ -32,4 +38,42 @@ export class JoinedGroupListComponent implements OnDestroy {
     if (sort) this.sort$.next(sort);
   }
 
+  onGroupLeaveClick(event: Event, group: JoinedGroup): void {
+    this.confirmationService.confirm({
+      target: event.target || undefined,
+      key: 'commonPopup',
+      message: $localize`Are you sure you want to leave this group?`,
+      header: $localize`Confirm Action`,
+      icon: 'pi pi-exclamation-triangle',
+      acceptLabel: $localize`Yes, leave group`,
+      accept: () => {
+        this.leaveGroup(group);
+      },
+      rejectLabel: $localize`No`,
+    });
+  }
+
+  leaveGroup(group: JoinedGroup): void {
+    const groupId = group.group.id;
+    const groupName = group.group.name;
+    this.groupLeaveService.leave(groupId)
+      .pipe(first())
+      .subscribe(() => {
+        this.refresh$.next();
+        this.messageService.add({
+          severity: 'success',
+          summary: $localize`Success`,
+          detail: $localize`You have left ${groupName}`,
+          life: TOAST_LENGTH,
+        });
```

Comment:
you'll have to get master (`merge master`) and use the new service.

Reply:
Fixed